File: 1894-merge-strings-alternately/merge-strings-alternately.py
class Solution:
    def mergeAlternately(self, word1: str, word2: str) -> str:
        i =0
        j = 0

        ans = ""

        while i < len(word1) or j < len(word2):
            if(i < len(word1)):
                ans += word1[i]
                i+=1
            if j < len(word2):
                ans += word2[j]
                j+=1
        return ans

        

File: 1894-merge-strings-alternately/merge-strings-alternately.cpp
class Solution {
public:
    string mergeAlternately(string word1, string word2) {
        int i=0;
        int j = 0;
        string ans;

        while(i < word1.size() && j < word2.size()){
           // cout<< "Ashish";
           cout<<word1[i]<<endl;
            ans.push_back(word1[i]);
            ans.push_back(word2[j]);
            i +=1;
            j +=1;
        }
        cout<<ans;
        while( i < word1.size()){
            ans.push_back(word1[i]);
            i+=1;
        }


        while(j < word2.size()){
           ans.push_back(word2[j]);
            j+=1;
        }
        cout << ans;
        return ans;
        
    }
};